### ISSUE TITLE
storage example doesn't work on loopback 3.x 

### ISSUE THREAD

**Issue by moshemal** [strongloop/loopback-example-storage#19]
### Bug or feature request

- [x ] Bug
- [ ] Feature request

### Description of feature (or steps to reproduce if bug)
1) Cloning and run loopback server of example-3.0: 

```
$ git clone https://github.com/strongloop/loopback-example-storage.git
$ cd loopback-example-storage/example-3.0
$ npm install
$ node .
```

2) open browser on http://localhost:3000/

### Expected result
list of files in the containers in the bottom of the page

### Actual result (if bug)
empty list with exception on the server side:

"Unhandled error for request GET /api/containers/container1/files: Error: remoting.context option was removed in version 3.0. See http://loopback.io/doc/en/lb2/Using-current-context.html for more details."

### Additional information (Node.js version, LoopBack version, etc)

node version 6.10.1
loopback version 3.6

**Comment by vkruoso** [comment]
Same here. The interface tells the files were uploaded but nothing is added to the storage folder and nothing is listed.

**Comment by ghost** [comment]
Possible duplicate: [#16](https://github.com/strongloop/loopback-example-storage/issues/16)

**Comment by b-admike** [comment]
Yes this is a duplicate of #16, and will be fixed by #17. Closing, but feel free to re-open if the issue persists after we land that PR.
